tests/js/list-tests.js

```diff
@@ -32,6 +32,9 @@ const DATE1 = new Date(1);
 const DATE2 = new Date(2);
 const DATE3 = new Date(3);
 
+const isNodeProcess = typeof process === 'object' && process + '' === '[object process]';
+const isElectronProcess = typeof process === 'object' && process.versions && process.versions.electron;
```

Comment:
I'm surprised to see that we haven't already moved these to `/tests/js/test-utils.js`.
Not saying that should be a part of this PR - just a note 🙂